lib/model/tripDatabase.dart

```diff
@@ -642,86 +642,147 @@ List<TripItem> generateTripItems() {
+          title: "Stacja XIII\nJezus z Krzyża zdjęty",
+          description:
+              """„O Święta Matko, miecz naprawdę przeniknął Twoją duszę. Jedynie przenikając duszę Matki, mógł dosięgnąć ciała Syna. Umarłemu Jezusowi, któremu nie mogła już zaszkodzić niemiłosierna włócznia, otworzyła bok, Tobie zaś przeszyła serce. Jego duszy już tam nie było, a Twoja nie mogła się stamtąd oderwać. Straszny ból przeszył Twoją duszę, toteż zupełnie słusznie nazywamy Cię więcej niż Męczennicą, ponieważ ból współcierpienia przewyższa udrękę ciała” (św. Bernard).\n\nNi ja ojca, matki, brata, ni żadnego przyjaciela;\nskądże pocieszenie mam mieć? Wolałabym stokroć umrzeć,\nniż widzieć żołnierza złego, co przebił bok Syna mego.""",
           previousStep: 11,
           nextStep: 13,
         ),
         StepItem(
-          imageAsset: "assets/images/trip_03.png",
-          title: "Stacja 14",
+          imageAsset: "assets/images/trip02/trip02_14.jpg",
+          caption: "Dąb szypułkowy (quercus robur)\nDrzewo to niebawem rozrośnie się potężnie jak dzieło Chrystusa. U Żydów panował zwyczaj chowania zmarłych pod dębami.",
+          title: "Stacja XIV\nJezus złożony do grobu",
+          description:
+              """„Śmierć nie oddziela nas od siebie, ponieważ wszyscy zdążamy tą samą drogą i odnajdziemy się w tym samym miejscu. Nie będziemy nigdy rozłączeni, ponieważ żyjemy dla Chrystusa i teraz jesteśmy złączeni z Chrystusem, idąc ku Niemu. Wszyscy razem będziemy kiedyś z Chrystusem” (św. Szymon z Tesalonik).\n\nMatki, co synaczki macie, ajako się w nich wy kochacie,\nkiedy wam z nich jeden umrze, ciężki ból ma serce wasze;\ncóż ja, com miała jednego, już nie mogę mieć inszego.""",
           previousStep: 12,
+          nextStep: 14,
+        ),
+        StepItem(
+          imageAsset: "assets/images/trip02/trip02_15.jpg",
+          caption: "Drobne sadzonki ostrokrzewu (ilex)\nOstrokrzew symbolizuje ojcostwo, a jako roślina wiecznie zielona -nieśmiertelność.",
+          title: "Stacja XV\nZmartwychwstanie Jezusa",
+          description:
+              """„Potrzebujemy małych i większych nadziei, które dzień po dniu podtrzymują nas w drodze. Jednak bez wielkiej nadziei, która musi przewyższać pozostałe, są one niewystarczające. Tą wielką nadzieją może być jedynie Bóg, który ogarnia wszechświat i który może nam zaproponować i dać to, czego sami nie możemy osiągnąć. Bóg jest fundamentem nadziei, nie jakikolwiek bóg, ale ten Bóg, który ma ludzkie oblicze i umiłował nas aż do końca: każdą jednostkę i ludzkość w całości” (papież Benedykt XVI).\n\nO, niestetyż, miły Panie, toć nie małe rozłączenie;\nprzedtem było miłowanie, a teraz ciężkie wzdychanie.\nCzemuż, Boże Ojcze, nie dbasz, o Synaczka pieczy nie masz?""",
+          previousStep: 13,
+          nextStep: 15,
+        ),
+        StepItem(
+          type: StepType.end,
+          title: "Zakończyłeś drogę krzyżową.",
```

Comment:
Perhaps better title would be "Odpust" or "Łaska odpustu" or "Zakończyłeś drogę krzyżową" (without period at the end)